### ISSUE TITLE
Twitter Sentiment Analysis NLP

### ISSUE THREAD

**Issue by Aadi71** [abhisheks008/DL-Simplified#100]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Twitter Sentiment Analysis** : </br>
:red_circle: **Aim**: Perform Sentiment Analysis with 3-4 Deep Learning Models and find the best-fit model.</br>
:red_circle: **Dataset**: https://www.kaggle.com/datasets/kazanova/sentiment140</br>
:red_circle: **Approach**: Try to use 3-4 algorithms to implement the models and compare all the algorithms to find the best-fitted model algorithm by checking the accuracy scores. Also do not forget to do an exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute to the Project:**
* You need to create a separate folder named the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project on other machines.
* Inside the `Model` folder, the `README.md` file must be filled up correctly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include the issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before starting Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue:**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open-source journey ahead. 😎

**Comment by Aadi71** [comment]
Hey, would like to work on this issue. Kindly assign it to me.

Full name: Aadi Jain
GitHub Profile Link: https://github.com/Aadi71
Email ID: jainaadi7102@gmail.com
Role: SSOC' 22 Participant

**Comment by abhisheks008** [comment]
Issue assigned to you @Aadi71

**Comment by abhisheks008** [comment]
Updates please @Aadi71

**Comment by Aadi71** [comment]
Working on it and will complete it by this week.

**Comment by abhisheks008** [comment]
@Aadi71 updates please.

**Comment by Aadi71** [comment]
I understand it got delayed by a huge time, but I will definitely try to complete it within the deadline.

**Comment by abhisheks008** [comment]
Try to do that, otherwise it'll of no use.

**Comment by Aadi71** [comment]
Closing this issue, I almost did it yet I cannot conclude.

**Comment by abhisheks008** [comment]
Do not close issues, let the other contributors to solve this issue. It'll be a opportunity for them too.

**Comment by arkyabanerjee** [comment]
Hey, would like to work on this issue. Kindly assign it to me.

Full name: Arkya Banerjee
GitHub Profile Link: github.com/arkyabanerjee
Email ID: banerjeearkya92@gmail.com
Role: SWOC S3 Participant

**Comment by abhisheks008** [comment]
Sorry for the inconvenience caused. Try to focus on DL techniques rather than ML algos. Issue assigned to you @arkyabanerjee. Go ahead!
